po/tr.po

```diff
@@ -0,0 +1,1471 @@
+# Turkish translation for se.sjoerd.Graphs
+# Copyright (C) 2023 se.sjoerd.Graphs' COPYRIGHT HOLDER
+# This file is distributed under the same license as the se.sjoerd.Graphs package.
+#
+# Sabri Ünal <libreajans@gmail.com>, 2023.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: se.sjoerd.Graphs\n"
+"Report-Msgid-Bugs-To: https://github.com/Sjoerd1993/Graphs/issues\n"
```

Comment:
Just a technicality here, when updating translations meson sets this as empty. Perhaps this has to be specified inside a build config somewhere as this affects other translations as well.